csv_handler/handlers.go

```diff
@@ -0,0 +1,67 @@
+// Add a new route, which receives a local file path in json body.
+// Handler method should read and parse the CSV.
+// And return the data in file in json format.
+// Import `./csv` in handlers.go and use the method as `csv.ParseCsv`
+
+package csvhandler
+
+import (
+	"github.com/abidkhan03/go_training/csv_handler/csv"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"github.com/go-chi/chi/v5"
+)
+
+type Request struct {
+	Path string `json:"path"`
+}
+
+type Response struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+}
+
+const PORT = "8000"
+
+func main() {
+	r := chi.NewRouter()
+	r.Post("/hello", handlers)
+	r.Post("/parse", ParseHandler)
+	fmt.Println("Server is running on port ", PORT)
+	http.ListenAndServe(PORT, r)
+}
```

Comment:
Why is there a `main` function in handlers?
Why `csv_handler` is a separate folder?
Why there is a single file for all kinds of handlers?